bc_obps/compliance/service/elicensing_invoice_service.py

```diff
@@ -470,12 +509,11 @@ def get_elicensing_invoice_for_dashboard(cls, user_guid: UUID) -> QuerySet[Elice
                 )
             )
 
-        for invoice in compliance_invoices:
-            _, _, total_fee, total_payments, total_adjustments = ElicensingInvoiceService.calculate_invoice_amount_due(
-                invoice
-            )
-            invoice.invoice_total = total_fee  # type: ignore[attr-defined]
-            invoice.total_payments = total_payments  # type: ignore[attr-defined]
```

Comment:
Having this after returning for internal users means the internal users previously wouldn't have gotten the calculated fields